compiler/rustc_builtin_macros/src/deriving/cmp/eq.rs

```diff
@@ -15,20 +15,15 @@ pub fn expand_deriving_eq(
     item: &Annotatable,
     push: &mut dyn FnMut(Annotatable),
 ) {
+    let span = cx.with_def_site_ctxt(span);
     let inline = cx.meta_word(span, sym::inline);
-    let no_coverage_ident =
-        rustc_ast::attr::mk_nested_word_item(Ident::new(sym::no_coverage, span));
-    let no_coverage_feature =
-        rustc_ast::attr::mk_list_item(Ident::new(sym::feature, span), vec![no_coverage_ident]);
-    let no_coverage = cx.meta_word(span, sym::no_coverage);
     let hidden = rustc_ast::attr::mk_nested_word_item(Ident::new(sym::hidden, span));
     let doc = rustc_ast::attr::mk_list_item(Ident::new(sym::doc, span), vec![hidden]);
-    let attrs = vec![
-        cx.attribute(inline),
-        cx.attribute(no_coverage_feature),
-        cx.attribute(no_coverage),
-        cx.attribute(doc),
-    ];
+    let mut attrs = vec![cx.attribute(inline), cx.attribute(doc)];
+    if cx.parse_sess().unstable_features.is_nightly_build() {
```

Comment:
This doesn't need to be conditional on `is_nightly_build`, does it?

Reply:
I'll try to change it and verify it still builds `core` and still disables use on beta/stable.

Reply:
Done